src/turret.cpp

```diff
@@ -490,9 +490,18 @@ npc vehicle::get_targeting_npc( vehicle_part &pt )
     cpu.name = string_format( pgettext( "vehicle turret", "The %s" ), pt.name().c_str() );
     // turrets are subject only to recoil_vehicle()
     cpu.recoil = 0;
+
     // These might all be affected by vehicle part damage, weather effects, etc.
-    cpu.set_skill_level( turret_query( pt ).base()->gun_skill(), 8 );
+    cpu.set_skill_level(skill_id("pistol"), 8);
+    cpu.set_skill_level(skill_id("rifle"), 8);
+    cpu.set_skill_level(skill_id("smg"), 8);
+    cpu.set_skill_level(skill_id("shotgun"), 8);
+    cpu.set_skill_level(skill_id("launcher"), 8);
+    cpu.set_skill_level(skill_id("archery"), 8);
+    cpu.set_skill_level(skill_id("throw"), 8);
```

Comment:
This enumeration will not work for modded turrets that use some weird skill.
It would be better if you just changed the `turret_query( pt ).base()->gun_skill()` into `pt.base.gun_skill()`.